.browserlistrc

```diff
@@ -0,0 +1,4 @@
+# Browsers that we support
```

Comment:
Use of a `.browserlistrc` is aligned with setup in other projects. "Recommended" approach (not sure why) is to add to `package.json`, but I prefer the separation. However anyone prefers, I can move this.

Also, these are only initial tests. Can be updated here and later.

Reply:
I know `browserlist` is easier to say, but is there a reason you went with that instead of `browserslist` (which matches the name of the config)?

Reply:
Confusion I guess I share with some others: https://github.com/browserslist/browserslist/issues/138

Fixed the name

Reply:
This is a logical OR, and results in the following list (`npx browserslist`)

```
and_chr 79
and_ff 68
and_qq 1.2
and_uc 12.12
android 76
baidu 7.12
bb 10
chrome 80
chrome 79
edge 80
firefox 72
ie 11
ie_mob 11
ios_saf 13.3
ios_saf 13.2
kaios 2.5
op_mini all
op_mob 46
opera 66
safari 13
samsung 10.1
```